deploy/docker/docker-compose.yml

```diff
@@ -51,6 +55,7 @@ services:
   dolphinscheduler-schema-initializer:
     image: ${HUB}/dolphinscheduler-tools:${TAG}
     env_file: .env
+    profiles: ["schema-init"]
     command: [ bin/create-schema.sh ]
```

Comment:
In ```create/upgrade-schema.sh``` will ```source  bin/dolphinscheduler-env.sh```. I think the ```.env``` file may not working in containers currently.

Reply:
> In `create/upgrade-schema.sh` will `source bin/env/dolphinscheduler-env.sh`. I think the `.env` file may not working in containers currently.

I found that `source bin/env/dolphinscheduler-env.sh` in `upgrade-scheme.sh` will not work because there is no such script file. So the .env file works.

Reply:
Oh....